content/slack/index.md

```diff
@@ -0,0 +1,48 @@
+# Slack
+
+Slack is a group chat app that serves as our group's virtual space for all sorts of mumblechatter. You can access Slack from your phone, desktop, or browser.
+
+{{< button href="/schedule" >}}Join Slack{{< /button >}}
+</br>
+</br>
+
+## Channels
+
+Conversations on Slack are organized into channels that start with `#`. The following sections share details about the most common. Don't see one you want? Ask in `#mumblechatter`. You can even [create your own channel](https://slack.com/help/articles/201402297-Create-a-channel), such as for your Shield Lock or a one-time event!
+
+### Default channels
+
+Everyone automatically is part of the following channels:
+
+| Channel | Description | 
+| --- | --- |
+| #1stF | Anything fitness-related, including "preblasts" or promotional info for who's leading each workout the night before.|
+| #2ndF | Anything fellowship-related, such as happy hours, spontaneous meetups to watch a game, or recommendations on things to do.|
+| #3rdF | Anything faith-related, such as praises, prayers, and other ways we can help each other out.|
+| #backblasts | A channel where backblasts, or reports for each workout of the day, are automatically added based on the Q filling out the **Paxmate** bot. You can also review the [Paxmate Dashboard](/paxmate-dashboard/) based on these reports. |
+| #cot-announcements | Announcements that you would like the pax to hear about in Circle of Trust at the end of each workout. These are usually upcoming events like a race or service project. Only admins can post to this channel. If you have an announcement, let someone on the [Leadership](/leadership/) team know.|
+| #mumblechatter | A catch-all channel for general questions, comments, observations of life, etc. Usually the most active channel on the site.|
+
+### Other channels of interest
+
+Channels are often used to share interests or coordinate activities around a certain type of event. As Peak City takes off, we are trying to take a "narrow and deep" approach to encourage activity (vs. a "wide and shallow" large number of channels with fewer participants).
```

Comment:
I thought keeping fewer channels would encourage denser posting in the beginning, so I tried to compress some of the popular ones into the following...not sure how many more we'd want. Maybe best to just remove and only describe default ones for now?

Reply:
I think the default ones can have a description in their header/description (in slack) so having them here is fine, but duplicated. However, this is a far more complete document giving an overview so I believe it's fine. We'll have to update the links.